pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -119,14 +118,17 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	kubeClient internalclientset.Interface,
 	volumePluginMgr *volume.VolumePluginMgr,
-	recorder record.EventRecorder) OperationExecutor {
+	recorder record.EventRecorder,
+	checkBinariesBeforeMount bool) OperationExecutor {
```

Comment:
Renaming sucks but I'm going to recommend `checkNodeCapabilitiesBeforeMount` instead of `checkBinariesBeforeMount` in all the files. It is more verbose, but it is less likely to become inaccurate: the checks you have at the moment only look for binaries, but other plugins may want to check for kernel modules or something else in the future.